src/app/tests/TestEventLogging.cpp

```diff
@@ -154,6 +154,16 @@ static void CheckLogReadOut(nlTestSuite * apSuite, chip::app::EventManagement &
     err = alogMgmt.FetchEventsSince(writer, clusterInfo, startingEventNumber, eventCount, chip::Access::SubjectDescriptor{});
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR || err == CHIP_END_OF_TLV);
 
+    auto eventTLVSize = writer.GetLengthWritten() / eventCount;
+    // XXX: Make sure that the sizes of our event storages are big enough to hold at least 3 events
+    //      but small enough not to hold 4 events. It is very important to check this because of the
+    //      hard-coded logic of this unit test.
+    //      The size of TLV-encoded event can vary depending on the UTC vs system time controlled by
+    //      the CHIP_DEVICE_CONFIG_EVENT_LOGGING_UTC_TIMESTAMPS, because the relative system time
+    //      will be most likely encoded on 1 byte, while the UTC time will be encoded on 8 bytes.
```

Comment:
```suggestion
    //      The size of TLV-encoded event can vary depending on the UTC vs system time controlled by
    //      the CHIP_DEVICE_CONFIG_EVENT_LOGGING_UTC_TIMESTAMPS, because the relative system time
    //      will be most likely encoded in 1 byte, while the UTC time will be encoded in 8 bytes.
```